client/post-editor/editor-ground-control/style.scss

```diff
@@ -38,15 +38,38 @@
 	margin: 0 4px 8px;
 }
 
-.editor-ground-control__publish-button {
+.editor-ground-control .editor-publish-button {
+	@extend .is-primary;
+
 	border-radius: 4px 0 0 4px;
 	flex-grow: 1;
+
+	@include breakpoint( "<660px" ) {
+		display: none;
+	}
 }
+
 .editor-ground-control__time-button {
-	border-radius: 0 4px 4px 0;
+	@include breakpoint( ">660px" ) {
+		@include is-primary;
```

Comment:
>`@extend` doesn't work inside `@media` queries. That was my first approach :(

Ah, well, that's a shame, though it at least appears to be an upcoming feature (sass/sass#1050).

Perhaps we'd be better to be more specific with the naming? A global mixin named `is-primary` doesn't seem exclusive to its application on a button (`button-is-primary` or `is-primary-button` come to mind).